modules/persistent-storage-csi-group-snapshots-create.adoc

```diff
@@ -0,0 +1,111 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-group-snapshots.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-group-snapshots-create_{context}"]
+= Creating a volume group snapshot
+
+When you create a `VolumeGroupSnapshot` object, {product-title} creates a volume group snapshot.
+
+.Prerequisites
+* Logged in to a running {product-title} cluster.
+* A persistent volume claim (PVC) created using a CSI driver that supports `VolumeGroupSnapshot` objects.
+* A storage class to provision the storage back end.
+* Administrator has created the `VolumeGroupSnapshotClass` object.
+* No pods are using the PVC that you want to take a snapshot of.
++
+[WARNING]
+====
+Creating a volume snapshot of a PVC that is in use by a pod can cause unwritten data and cached data to be excluded from the snapshot. To ensure that all data is written to the disk, delete the pod that is using the PVC before creating the snapshot.
+====
+
+.Procedure
+
+To create a volume group snapshot:
+
+. Locate (or create) the PVCs that you want to include in the volume group snapshot:
++
+[source, terminal]
+----
+$ oc get pvc
+
+NAME        STATUS    VOLUME                                     CAPACITY   ACCESSMODES   AGE
+pvc-0       Bound     pvc-a42d7ea2-e3df-11ed-b5ea-0242ac120002   1Gi        RWO           48s
+pvc-1       Bound     pvc-a42d81b8-e3df-11ed-b5ea-0242ac120002   1Gi
+----
++
+This example uses two PVCs
+
+. Label the PVCs to belong to a snapshot group by running the following commands:
++
+[source, terminal]
+----
+$ oc label pvc pvc-0 group=myGroup
+persistentvolumeclaim/pvc-0 labeled
+
+$ oc label pvc pvc-1 group=myGroup
+persistentvolumeclaim/pvc-1 labeled
+----
++
+In this example, you are labeling PVC "pvc-0" and "pvc-1" to belong to group "myGroup".
+
+. Create a `VolumeGroupSnapshot` object to specify your volume group snapshot by running the following command with the example file:
++
+[source,terminal]
+----
+$ oc create -f <volume-group-snapshot-filename>.yaml
+----
++
+.Example VolumeGroupSnapshot YAML file
+[source, yaml]
+----
+apiVersion: groupsnapshot.storage.k8s.io/v1alpha1
+kind: VolumeGroupSnapshot <1>
+metadata:
+  name: <volume-group-snapshot-name> <2>
+  namespace: <namespace> <3>
+spec:
+  volumeGroupSnapshotClassName: <volume-group-snapshot-class-name> <4>
+  source:
+    selector:
+      matchLabels:
+        group: myGroup <5>
+----
+<1> The `VolumeGroupSnapshot` object requests creation of a volume group snapshot for multiple PVCs.
+<2> Name of the volume group snapshot.
+<3> Namespace for the volume group snapshot.
+<4> The `VolumeGroupSnapshotClass` name. This object is created by the administrator and describes how volume group snapshots should be created.
+<5> The name of the label used to group the desired PVCs for the snapshot. In this example, it is "myGroup".
+
+.Results
```

Comment:
I am not sure if we use something called Results. We could have any verification queries which will validate the success of procedure.